packages/validator/src/cli-validator/run-validator.js

```diff
@@ -254,4 +252,37 @@ function logError(description, message = '') {
   }
 }
 
+function handleSpectralError(error) {
+  logError('There was a problem with spectral.', getError(error));
+  logger.error('Additional error details:');
+
+  // The nimma errors are especially difficult to understand, so we do some parsing
+  // based on the expected structure of the errors to extract the error cause and
+  // the specific file/line number the problem occurs on.
+  if (error.message && error.message === 'Error running Nimma') {
+    if (error instanceof AggregateError) {
+      error.errors.forEach(err => {
+        if (err.cause && err.cause.cause) {
+          const { cause } = err.cause;
+          const errInfo = {};
+          try {
+            // Look for the filepath/line number at the top of the stack.
+            errInfo.topOfStack = cause.stack.split(' at ')[1];
+            errInfo.reason = cause.message;
+          } catch (e) {
+            logger.debug('Could not parse Spectral error information');
+            logger.debug(e.message);
+            logError(err);
+          } finally {
+            logger.error(`Cause: ${errInfo.reason}`);
```

Comment:
Do we want the logError() call from the catch() AND the two logger.error() statements to be executed in the event of an exception while parsing error info?
Just checking...

Reply:
Good catch! I wasn't thinking. I didn't mean to use `finally`, I meant to make those error logs conditional on NOT throwing an exception. I'll make that update before merging

Reply:
yeah, I think just putting them inside the try block and removing the finally block would do the trick.